journal.py

```diff
@@ -129,10 +130,21 @@ def _get_open_close_moves(cls, _type, description, fiscalyear, accounts,
         sequence = Sequence.search([
                 ('id', '=', move.period.post_move_sequence_used.id),
                 ])[0]
-        sequence_prefix = Sequence._process(sequence.prefix, date=move.date)
-        sequence_sufix = Sequence._process(sequence.suffix, date=move.date)
+        sequence_prefix = Sequence._process(sequence.prefix, date=move.post_date)
+        sequence_sufix = Sequence._process(sequence.suffix, date=move.post_date)
```

Comment:
The field to be in consideration is the "date" not the "post_date".

Reply:
> The field to be in consideration is the "date" not the "post_date".

Examples moves with date, post_date and post_number:

```
id   |    date    | post_date  | post_number  
--------+------------+------------+--------------
 135498 | 2023-01-01 | 2022-12-21 | AS2022_00001
 135750 | 2023-01-02 | 2022-12-23 | AS2022_00002

   id   |    date    | post_date  | post_number  
--------+------------+------------+--------------
 162372 | 2024-01-01 | 2023-12-21 | AS2023_00001
```
 
 Also, in case the post_number in account.move is not same pattern from the sequence (because change configuration sequence after post the move), try to extract the number.

For this case, see ```re.find()``` in the code review.